lib/tapioca/cli.rb

```diff
@@ -19,10 +19,6 @@ class Cli < Thor
                   aliases: ["--out", "-o"],
                   banner: "directory",
                   desc: "The output directory for generated RBI files"
-    class_option :generate_command,
-                  aliases: ["--cmd", "-c"],
-                  banner: "command",
-                  desc: "The command to run to regenerate RBI files"
```

Comment:
I suggest that we still leave the option in place and show a warning to users if they are using the option. It would be better than making them fail weirdly on CI or something, in case they are using this.

Reply:
In other words, deprecate now with removed functionality and then remove the option in a later version.